src/NLog/Targets/ConsoleTarget.cs

```diff
@@ -147,6 +164,31 @@ protected override void CloseTarget()
             base.CloseTarget();
         }
 
+        private static bool IsConsoleAvailable()
```

Comment:
Fair point. I noticed that non trivial code has been duplicated. We always prefer reuse above duplication